osquery/config/plugins/tests/tls_config_tests.cpp

```diff
@@ -9,17 +9,36 @@
  */
 
 #include <gtest/gtest.h>
+#include <vector>
```

Comment:
Place these C++ includes above our local dependencies include, and put a space between the groupings please.